IPv6_test.go

```diff
@@ -30,9 +30,10 @@ func Test_ParseIPv6(t *testing.T) {
 		{"::1:2:3:4",       0x0000000000000000, 0x0001000200030004, false},
 		{"::1:2:3:4:5",     0x0000000000000001, 0x0002000300040005, false},
 		{"::1:2:3:4:5:6",   0x0000000000010002, 0x0003000400050006, false},
-		{"::1:2:3:4:5:6:7", 0x0000000100020003, 0x0004000500060007, false},
+		{"::1:2:3:4:5:6:7", 0x0000000100020003, 0x0004000500060007, false}, // invalid but accepted; see RFC 5952 section 4.2.2
 		{"fec0", 0, 0, true},
 		{"fec0:::1", 0, 0, true},
+		{"fec0::3:4:5:6:7:8", 0, 0, true}, // invalid but accepted; see RFC 5952 section 4.2.2
```

Comment:
this is a bad test. per the comment, the address is invalid but accepted so the test should pass. however, a test like this would also effectively be a duplicate of line 33.

Reply:
Good point. I'll tweak the PR.